packages/workbox-webpack-plugin/src/inject-manifest.js

```diff
@@ -113,8 +116,13 @@ class InjectManifest {
       .map(JSON.stringify)
       .join(', ');
 
-    const postInjectionSWString = `importScripts(${importScriptsString});
+    const setConfigString = modulePathPrefix
+      ? `workbox.setConfig({modulePathPrefix: ` +
+        `${JSON.stringify(modulePathPrefix)}});`
```

Comment:
Do you think `JSON.stringify` is necessary?

Reply:
It's safer than wrapping the value in either `"..."` or `'...'`, just on the off chance modulePathPrefix contains any quote characters. (Not that it should, but still.)

`JSON.stringify()` is guaranteed to return a properly escaped string.